legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -496,6 +517,16 @@ def alter_outputs(filing: FilingStorage, outputs: set):
         elif filing.filing_type == 'dissolution' and filing.filing_sub_type == 'administrative':
             # Supress Certificate of Dissolution for Admin Dissolution
             outputs.remove('certificateOfDissolution')
+        elif filing.filing_type == 'correction':
+            corrected_filing_id = filing.filing_json.get('correction', {}).get('correctedFilingId')
+            original_filing = FilingStorage.find_by_id(corrected_filing_id)
+            if FilingMeta.is_special_resolution_correction(
+                    business.legal_type, filing.filing_json['filing'], business, original_filing
+                    ):
+                outputs.add('certifiedRules')
```

Comment:
Had to put it here, otherwise other correction filings would fail for co-ops. Co-ops AR would fail.